src/features/click-to-load.js

```diff
@@ -24,8 +24,20 @@ const entityData = {}
 // Used to avoid displaying placeholders for the same tracking element twice.
 const knownTrackingElements = new WeakSet()
 
```

Comment:
I'm getting this error when toggling on/off the previews:
![image](https://user-images.githubusercontent.com/14364481/230443128-cc20e143-692d-45e1-a553-265ca70c3e92.png)

Reply:
I've not been able to reproduce that, did rebuilding with latest extension (and tracker-surrogates) code help?

Reply:
_(For the record, we discussed this in MatterMost and after fetching the latest node_modules this issue went away.)_